bitcoin/src/blockdata/script/mod.rs

```diff
@@ -84,6 +85,9 @@ pub use self::{
     push_bytes::*,
 };
 
+/// The default maximum size of scriptints.
+const DEFAULT_MAX_SCRIPTINT_SIZE: usize = 4;
```

Comment:
The purpose of this constant is that

* All script opcodes take maximum 4-byte numbers as inputs.
* *Except* CLTV which takes a maximum 5-byte number.

Personally I think it'd be clearer if we just had a `read_scriptint` and `read_cltv_scriptint` method which each passed into a `read_scriptint_internal` and hardcoded the 4 and 5. It's a minor refactor relative to what's here but I'd find it easier to follow if everything was together and we didn't have a new constant to memorize the value of.